lib/appenders/mailgun.js

```diff
@@ -17,11 +16,12 @@ function mailgunAppender(_config, _layout) {
             to: _config.to,
             subject: _config.subject,
             text: layout(loggingEvent, config.timezoneOffset)
-        };
+        }
 
         mailgun.messages().send(data, function (error, body) {
-            if (error != null) console.error('log4js.mailgunAppender - Error happened', error);
-        });
+            if (error != null) console.error("log4js.mailgunAppender - Error happened", error);
+            else console.log("Success", body);
```

Comment:
You don't really want things going to `console.log` every time you send something via this appender, do you?